xml/ha_yast_cluster.xml

```diff
@@ -65,6 +65,10 @@
       <systemitem>bindnetaddr</systemitem> to
       <literal>192.168.5.64</literal>.
      </para>
+     <para> If <systemitem>nodelist</systemitem> with
+       <systemitem>ringX_addr</systemitem> is explicitly configured in
+       <filename>/etc//corosync/corosync.conf</filename>,
```

Comment:
There seems to be an extra '/' between "etc" and "corosync"? :-)

Reply:
Ahh, right! Thanks! Fixed it.